docs/design/release_automation_design.md

```diff
@@ -0,0 +1,45 @@
+## Release Automation Process Design
+
+### Abstract
+
+The release automation process is an automated system that will handle Github
+releases. What this entails is tagging a release, ensure that the release blurb
+contains all the changes that are to go out, and potentially handle release
+artifacts.
+
+### Specification
+
+The release automation process will be triggered by a periodic AWS Lambda
+(Lambda) job that will signify whether or not regeneration of the SDK needs to
+occur and/or tag a Github release.
+
+Regeneration of the SDK will be determined by diffing the swagger.yaml files in
+the SDK and Firecracker. If there is a difference in the swagger files, this
+will regenerate the SDK and also tag a new release. In the event that there are
```

Comment:
What do you mean by `will regenerate the SDK` ? Will this publish new release to Github?